hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/TestDirListingMetadata.java

```diff
@@ -130,7 +133,8 @@ public void testGet() {
     PathMetadata pathMeta2 = new PathMetadata(
         new S3AFileStatus(true, new Path(path, "dir2"), TEST_OWNER));
     PathMetadata pathMeta3 = new PathMetadata(
-        new S3AFileStatus(123, 456, new Path(path, "file1"), 8192, TEST_OWNER));
+        new S3AFileStatus(123, 456, new Path(path, "file1"), 8192, TEST_OWNER,
+            "abc", "def"));
```

Comment:
Why not use TEST_ETAG and TEST_VERSIONID here?

Reply:
Oversight.  Will fix.

Reply:
Addressed in https://github.com/apache/hadoop/pull/675/commits/90d5c9c16277bed3deb7c6ff760d875c1123fe83